src/enforce/rules/helpers/expect_type/spec.js

```diff
@@ -25,12 +23,14 @@ const noThrow = {
 
 describe('Tests expect type helper', () => {
 
+    const throwString = (...args) => `[Passable]: Failed to execute '${args[0]}': expected ${args[1]} to be a ${args[2]}`;
```

Comment:
If this error structure has a usage other than this, or will have, perhaps we should move it to `helpers.js`.

Reply:
At the moment this is the only use case. Once (and if) any is added, a move should indeed be considered.